tests/1d_flow_damped.prm

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------
+#
+# Copyright (C) 2013 - 2016 by the adaflo authors
```

Comment:
Could you also upload the output. Btw. where is the actual test? This is only the parameter file!?

Reply:
I've added the output file.

Reply:
Don't you also have `tests/1d_flow_damped.cc`?

Reply:
No. I simply usw '1d_flow.cc'.

Reply:
I see :)